src/System.Private.ServiceModel/pkg/System.Private.ServiceModel.pkgproj

```diff
@@ -3,14 +3,22 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
+    <!-- Ideally we'd harvest this from the runtime dependencies -->
+    <Version>4.0.1</Version>
     <PreventImplementationReference>true</PreventImplementationReference>
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\src\System.Private.ServiceModel.builds">
-      <AdditionalProperties>FilterToOSGroup=Windows_NT</AdditionalProperties>
-      <UndefineProperties>%(ProjectReference.UndefinedProperties);PackageTargetFramework</UndefineProperties>
-    </ProjectReference>
+    <!-- Implementation only package -->
+    <File Include="$(PlaceholderFile)">
+      <TargetPath>ref/dnxcore50</TargetPath>
+    </File>
+    <File Include="$(PlaceholderFile)">
+      <TargetPath>ref/netcore50</TargetPath>
+    </File>
```

Comment:
You can omit these ones.  We inject a common one with PreventImplementationReference.